src/reference/modules/installation/pages/upgrade/notes.adoc

```diff
@@ -9,6 +9,39 @@ xref:administration:procedures.adoc#_migration_backups_and_restores[backup proce
 
 ====
 
+== Plugins disabled during upgrade
+
+Each time Rudder is upgraded, plugins are disabled. You need to enable them 
+back after upgrade.
+For that, start by checking if update are available for you new version (it will
+be mandatory for a minor or major version upgrade), and enable them back:
+
+----
+
+rudder package update
```

Comment:
This only work for people with a subscription, we need an alternate doc for other users.